liam/files/js.js

```diff
@@ -1,12 +1,115 @@
-console.log(screen.width)
+var even = false;
 
 window.addEventListener("load", (a) => {
-    const doodle = document.getElementById("doodle");
-    doodle.setAttribute("width", Math.min("900", screen.width) + "px");
-    const bg = document.getElementById("background-player");
-    var s = (screen.width / 1717)*1920;
-    bg.style = `width: ${Math.max(s, screen.height * (1920/1080))}px; height: ${Math.max(screen.height, s * (1080/1920))}px; margin-left:-100px`;
```

Comment:
We needed this to get rid of the black bars. Are they still gone now?

Reply:
the video is exactly full-hd, so when run in full screen there will be no bars and you'll see the entire thing

Reply:
![image](https://user-images.githubusercontent.com/5946409/39176764-e49c4c08-47ad-11e8-8c01-8fe4e8852d58.png)
Nope, they are back

Reply:
Then you're not in full screen.
Anyway we're going to see if we can get the mp4 and run it directly to avoid the iframe which causes the black bars.